apps/server/src/modules/oauth-provider/mapper/oauth-provider-response.mapper.spec.ts

```diff
@@ -1,14 +1,141 @@
 import { OauthProviderResponseMapper } from '@src/modules/oauth-provider/mapper/oauth-provider-response.mapper';
+import {
+	ProviderConsentResponse,
+	ProviderOauthClient,
+	ProviderOidcContext,
+	ProviderRedirectResponse,
+} from '@shared/infra/oauth-provider/dto';
+import { RedirectResponse } from 'apps/server/src/modules/oauth-provider/controller/dto/response/redirect.response';
+import { ConsentResponse } from '@src/modules/oauth-provider/controller/dto/response/consent.response';
+import { OauthClientResponse } from '@src/modules/oauth-provider/controller/dto/response/oauth-client.response';
+import { OidcContextResponse } from '@src/modules/oauth-provider/controller/dto/response/oidc-context.response';
+
+class OauthProviderResponseMapperSpec extends OauthProviderResponseMapper {
```

Comment:
We only want to test the protected method mapOidcContextResponse() and use inheritance for this.
Otherwise we could of course also test this in another test. But personally I think it's a little nicer